index.js

```diff
@@ -213,23 +216,36 @@ async function checkHeartbeat() {
 
   for (const location of locations) {
     let latestRadar
+    let latestDoor
+
     try {
       if (location.radarType === RADAR_TYPE.XETHRU) {
         const xeThruData = await redis.getLatestXeThruSensorData(location.locationid)
-        latestRadar = moment(xeThruData.timestamp, 'x')
+        latestRadar = convertToSeconds(xeThruData.timestamp)
       } else if (location.radarType === RADAR_TYPE.INNOSENT) {
         const innosentData = await redis.getLatestInnosentSensorData(location.locationid)
-        latestRadar = moment(innosentData.timestamp, 'x')
+        latestRadar = convertToSeconds(innosentData.timestamp)
       }
-      const currentTime = moment()
-      const radarDelay = currentTime.diff(latestRadar)
-
-      if (radarDelay > RADAR_THRESHOLD_MILLIS && !location.heartbeatSentAlerts) {
-        helpers.logSentry(`Radar Heartbeat threshold exceeded; sending alerts for ${location.locationid}`)
+      const doorData = await redis.getLatestDoorSensorData(location.locationid)
+      latestDoor = convertToSeconds(doorData.timestamp)
+      const redisTime = await redis.getCurrentTimeinSeconds()
+      const radarDelay = redisTime - latestRadar
+      const doorDelay = redisTime - latestDoor
+
+      const doorHeartbeatExceeded = doorDelay > helpers.getEnvVar('DOOR_THRESHOLD_SECONDS')
+      const radarHeartbeatExceeded = radarDelay > helpers.getEnvVar('RADAR_THRESHOLD_SECONDS')
```

Comment:
I like that you changed this to seconds. I think it makes more sense. However, the names of the environment variables in `.env.example` also need to change to `_SECONDS` then.

Reply:
yes good catch